metadata-ingestion/src/datahub/ingestion/source/looker/lookml_concept_context.py

```diff
@@ -382,14 +388,62 @@ def _get_list_dict(self, attribute_name: str) -> List[Dict]:
             return ans
         return []
 
+    def _include_parent_fields(
+        self, child_fields: List[dict], field_type: str
+    ) -> List[Dict]:
+        # Fetch the fields from the parent view, i.e., the view name mentioned in view.extends, and include those
+        # fields in child_fields. This inclusion will resolve the fields according to the precedence rules mentioned
+        # in the LookML documentation: https://cloud.google.com/looker/docs/reference/param-view-extends
+
+        parent_fields: Optional[Any] = self.get_including_extends(
+            field=field_type,
+            extends_only=True,
+        )
+
+        if parent_fields is None:
+            return child_fields  # No parent fields found
+
+        # Create a map field-name vs field
+        child_field_map: dict = {}
+        for field in child_fields:
+            assert (
+                NAME in field
+            ), "A lookml view must have a name field"  # name is required field of lookml field array
+
+            child_field_map[field[NAME]] = field
+
+        for field in parent_fields:
+            assert (
+                NAME in field
+            ), "A lookml view must have a name field"  # name is required field of lookml field array
+
+            if field[NAME] in child_field_map:
+                # This is an override case where the child has redefined the parent field.
+                # There are some additive attributes; however, we are not consuming them in metadata ingestion
+                # and hence not adding them to the child field.
+                continue
+
+            child_fields.append(field)
```

Comment:
does it make sense to extract the mapping merge logic into a helper method?

Reply:
Yup